ci/travis/ci.sh

```diff
@@ -110,7 +110,7 @@ upload_wheels() {
 test_python() {
   if [ "${OSTYPE}" = msys ]; then
     # Increased timeout from default of timeout=300 due to test_basic
-    bazel test -k --config=ci --test_timeout=600 --build_tests_only -- \
+    bazel test -k --test_timeout=600 --build_tests_only -- \
```

Comment:
I think `bazel test --config=ci` should be kept so when looking at these script, developers can use these command to reproduce it on their local machine.

Reply:
Hmm okay I added it back.